straxen/plugins/merged_s2s/merged_s2s.py

```diff
@@ -141,8 +168,11 @@ def compute(self, peaklets, lone_hits):
 
         strax.compute_widths(merged_s2s)
 
-        if n_top_pmts_if_digitize_top <= 0:
-            merged_s2s = drop_data_top_field(merged_s2s, self.dtype, "_drop_top_merged_s2s")
+        # if n_top_pmts_if_digitize_top <= 0:
```

Comment:
Why these two lines are commented out?

Reply:
If we set `sum_waveform_top_array` as `False`, there might be a bug. I have not tested, though.

Reply:
I removed the lines.